tests/test_pagination.py

```diff
@@ -9,6 +9,14 @@ class LinkedinAdsPaginationTest(TestLinkedinAdsBase):
     def name():
         return "tap_tester_linkedin_ads_pagination_test"
 
+    def get_properties(self, original: bool = True):
+        """Returns the connection properties"""
+        return {
+            "start_date" : "2018-08-21T00:00:00Z",
+            "accounts": "503491473,502890776",
+            "page_size": 100
+        }
+
```

Comment:
Also if we just wanna change the start date we can change the self.start_date and use `connections.ensure_connection(self, original_properties=False)` to use it